FEXCore/Source/Interface/Core/OpcodeDispatcher.cpp

```diff
@@ -93,8 +88,9 @@ void OpDispatchBuilder::SyscallOp(OpcodeArgs) {
     DefaultSyscallFlags = FEXCore::IR::SyscallFlags::NORETURNEDRESULT;
   }
   else if (OSABI == FEXCore::HLE::SyscallOSABI::OS_WIN32) {
-    NumArguments = 2;
-    GPRIndexes = &GPRIndexes_Win32;
+    // Since the whole context is going to be saved at entry anyway, theres no need to do additional work to pass in args
+    NumArguments = 0;
+    GPRIndexes = nullptr;
```

Comment:
Was just looking at this piece of code. I think there's an issue here because we are setting GPRIndexes to null but the next thing we do is to dereference GPRIndexes. I don't understand the WIN32 context but should we have an early exit here?

Reply:
This does end up creating a null reference, however it's never accessed since NumArguments is 0, will probably be cleaner to just keep GPRIndexes as a pointer and avoid the UB